rpaas/nginx/configuration_render.go

```diff
@@ -135,6 +135,12 @@ http {
     vhost_traffic_status_zone;
 {{end}}
 
+{{with .Instance.Host}}
+    upstream rpaas_backend_default {
```

Comment:
In the situation I described earlier we could have multiple upstreams here by iterating on `.Instance.Binds`. `upstream rpaas_backend_default` could be set to `.Instances.Binds[0].Host` for compatibility and additional upstreams would be created following some name convention like `upstream rpaas_backend_{{.Name}}`